### ISSUE TITLE
Replace fitText function

### ISSUE THREAD

**Issue by campos20** [thewca/tnoodle#306]
We have 2 fitText functions, working in different ways. It would be nice to replace the [old one](https://github.com/thewca/tnoodle/blob/master/webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java#L714) and some [code related](https://github.com/thewca/tnoodle/blob/master/webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java#L864) with the [new one](https://github.com/campos20/tnoodle/blob/master/webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java#L774).

**Comment by jfly** [comment]
How do you plan to replace splitTextToLineChunks? Did you know we use it to implement the alternate row highlighting? See https://github.com/thewca/tnoodle/blob/8bd6dcc4f409baa6697d6f46891b574460d97e1c/webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java#L1034-L1045

**Comment by campos20** [comment]
I did not know, but this type of issue will came up when the work start. If there's to much side code depending on it, it's going to be kept. Anyways, since I started working on the table for FMC sheet, I've been considering to use a table for this highlight as well, if it slims down the code or reduce [hacks](https://github.com/thewca/tnoodle/blob/master/webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java#L967).
